xlators/mgmt/glusterd/src/glusterd-volgen.c

```diff
@@ -3933,6 +3934,12 @@ set_volfile_id_option(volgen_graph_t *graph, glusterd_volinfo_t *volinfo,
         if (ret)
             goto out;
 
+        if (!dict_get_sizen(volinfo->dict, "cluster.use-anonymous-inode")) {
```

Comment:
I think for use-anonymous-inode you can implement a new function and put below check instead of 
checking conf->op_version.

If (glusterd_get_op_version_for_key(cluster.use-anonymous-inode) > volinfo->client_op_version)
    return 0;

If a client op_version behind of op_version mentioned in glusterd_volopt_map table then we don't need to enable the option on the graph.

Reply:
I have tried below check instead of above code-change, it will work fine.
if (glusterd_get_op_version_for_key("cluster.use-anonymous-inode") <= volinfo->client_op_version) {
+            ret = xlator_set_fixed_option(xlator, "use-anonymous-inode", "yes");
+            if (ret)
+                goto out;
+        }

Reply:
Okay, I will test this out over the weekend and refresh.